.github/workflows/cifuzz.yml

```diff
@@ -0,0 +1,35 @@
+name: CIFuzz
+on: [pull_request]
```

Comment:
It's often useful to be able to run a workflow manually, for testing. Here I would add `workflow_dispatch`:

```suggestion
on: 
  pull_request:
  workflow_dispatch:
```